save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -0,0 +1,214 @@
+/**
+ * A view with owner details
+ */
+
+package org.cqfn.save.frontend.components.views
+
+import org.cqfn.save.domain.ImageInfo
+import org.cqfn.save.entities.Project
+import org.cqfn.save.frontend.components.basic.privacySpan
+import org.cqfn.save.frontend.components.tables.tableComponent
+import org.cqfn.save.frontend.utils.*
+
+import csstype.Left
+import csstype.Position
+import csstype.TextAlign
+import csstype.Top
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.asList
+import org.w3c.fetch.Headers
+import org.w3c.xhr.FormData
+import react.*
+import react.dom.*
+import react.table.columns
+
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.html.InputType
+import kotlinx.html.hidden
+import kotlinx.html.js.onChangeFunction
+
+/**
+ * `Props` retrieved from router
+ */
+@Suppress("MISSING_KDOC_CLASS_ELEMENTS")
+external interface OrganizationProps : PropsWithChildren {
+    var organizationName: String
+}
+
+/**
+ * [State] of project view component
+ */
+external interface OrganizationViewState : State {
+    /**
+     * Flag to handle uploading a file
+     */
+    var isUploading: Boolean
+
+    /**
+     * Image to owner avatar
+     */
+    var image: ImageInfo?
+}
+
+/**
+ * A Component for owner view
+ */
+class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(false) {
+    init {
+        state.isUploading = false
+    }
+
+    override fun componentDidMount() {
+        super.componentDidMount()
+        GlobalScope.launch {
+            val avatar = getAvatar()
+            setState {
+                image = avatar
+            }
+        }
+    }
+
+    @Suppress("TOO_LONG_FUNCTION", "LongMethod", "MAGIC_NUMBER")
+    override fun RBuilder.render() {
+        div("d-sm-flex align-items-center justify-content-center mb-4") {
+            h1("h3 mb-0 text-gray-800") {
+                +props.organizationName
+            }
+        }
+
+        div("row justify-content-center") {
+            // ===================== LEFT COLUMN =======================================================================
+            div("col-2 mr-3") {
+                div("text-xs text-center font-weight-bold text-primary text-uppercase mb-3") {
+                    +"Organization"
+                }
+
+                div {
+                    attrs["style"] = kotlinext.js.jso<CSSProperties> {
```

Comment:
```suggestion
                    attrs["style"] = kotlinx.js.jso<CSSProperties> {
```
Did you see deprecation warning in IDEA?

Reply:
Fixed